src/modules/workspace/abtest/start.ts

```diff
@@ -56,14 +57,23 @@ ${chalk.red(significanceLevel)} significance level. Proceed?`,
 
 
 export default async () => {
-  await checkIfABTesterIsInstalled()
+  const abTesterManifest = await installedABTester()
   const workspace = await promptProductionWorkspace('Choose production workspace to start A/B test:')
+
+  if (semver.satisfies(abTesterManifest.version, '>=0.10.0')) {
+    log.info(`Setting workspace ${chalk.green(workspace)} to A/B test`)
+    await abtester.start(workspace)
```

Comment:
*nit*: as a simple client, `abtester` doesn't treat errors, just throw them. If the abtest app has specific errors which can be treated by toolbelt with better error messages, remember to take of them when you use the `start`, `startLgeacy` and `finish` methods.